.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -29,11 +25,13 @@ Please fill this for deployment related issues:
 - is conda or virtualenv used?: 
 - is running in docker?:
 
+**Log message**:
+ - nnimanager.log: 
+ - dispatcher.log:
+ - nnictl stdout and stderr:
+ 
+reference for these file: [log](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director), [stdout/stderr](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout)
```

Comment:
Where can you find the log files: https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director